partner_event/models/event_registration.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2014 Tecnativa S.L. - Pedro M. Baeza
+# Copyright 2015 Tecnativa S.L. - Javier Iniesta
+# Copyright 2016 Tecnativa S.L. - Antonio Espinosa
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2020 Tecnativa S.L. - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = "event.registration"
+
+    partner_id = fields.Many2one(ondelete="restrict")
+    attendee_partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Attendee Partner",
+        ondelete="restrict",
+        copy=False,
+    )
+
+    def _prepare_partner(self, vals):
+        return {
+            "name": vals.get("name") or vals.get("email"),
+            "email": vals.get("email", False),
+            "phone": vals.get("phone", False),
+        }
+
+    @api.model
```

Comment:
It'd be nice to pass it to `@api.model_create_multi`

Reply:
Some example addon that uses `@api.model_create_multi` to know what things should be changed and / or how to change them?

Reply:
Sure :)

https://www.odoo.com/documentation/13.0/reference/orm.html#odoo.api.model_create_multi
A simple example:
https://github.com/odoo/odoo/blob/054d4bc6bc219bcc6b0a64265e8d7e9c7423dbc8/addons/uom/models/uom_uom.py#L96-L102

Reply:
Thanks for the example, uploaded changes.